pkg/cri/config/config.go

```diff
@@ -329,6 +329,8 @@ type PluginConfig struct {
 	// The string is in the golang duration format, see:
 	//   https://golang.org/pkg/time/#ParseDuration
 	ImagePullProgressTimeout string `toml:"image_pull_progress_timeout" json:"imagePullProgressTimeout"`
+	// ImagePullWithTransferService specifies whether to pull image using transfer service otherwise it uses client library
```

Comment:
fyi.. https://github.com/containerd/containerd/blob/main/docs/cri/config.md?plain=1#L225-L228 these descriptions go in the cri detailed config description..  

grr it's missing imagepullproresstimeout :)

Reply:
Good catch.  I will add missed configs there.

Reply:
Done